project/MiMaFilters.scala

```diff
@@ -4,28 +4,5 @@ import com.typesafe.tools.mima.core.ProblemFilters._
 
 object MiMaFilters {
   val Library: Seq[ProblemFilter] = Seq(
-    // New APIs marked @experimental in 3.0.1
-    exclude[DirectMissingMethodProblem]("scala.quoted.Quotes.valueOrAbort"),
```

Comment:
I thought we should remove those in `3.1.0`

Reply:
@sjrd is this the expects behavior of MiMa in our setup?

Reply:
Because of my other comments, all of these will have to restored.

That said, once we do bump to 3.0.1, we will be able to drop them. Typically we use MiMa exclusions with a "transitive" mindset: if 3.0.1 was fine, and the changes since 3.0.1 are fine, then the current state is fine.

It is also possible to configure MiMa to check against a *set* of previous releases, but that is usually overkill.